src/content/apis/SMS/telesign SMS Verify .md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Can you please update the file names as per the other files that are available.
And also please upload the file under `miscs` directory